buildpack.toml

```
@@ -1,12 +1,12 @@
 api = "0.10"
 
 [buildpack]
-id = "heroku/debian-packages"
+id = "heroku/deb-packages"
 version = "0.0.1"
-name = "Heroku Debian Packages Buildpack"
-description = "Heroku's buildpack for installing Debian packages."
-homepage = "https://github.com/heroku/buildpacks-debian-packages"
-keywords = ["debian", "packages"]
+name = "Heroku .deb Packages Buildpack"
+description = "Heroku's buildpack for installing .deb packages."
```

```suggestion
description = "Heroku's buildpack for installing .deb system packages."
```

A hint for someone wanting to install system dependencies/packages but isn't explicitly looking for "deb" packages.